File: example/macos/Runner/Classes/tools/Tools.m
#import "Tools.h"
#define fileManager [NSFileManager defaultManager]
@implementation Tools

//Log
+ (void)log:(id)info{
    NSLog(@"CuriosityLog = %@", info);
}

+ (NSString *)resultInfo:(NSString *)info{
    return [NSString stringWithFormat:@"Curiosity:%@",info];
}

+ (NSString *)resultFail{
    return @"Curiosity:fail";
}
+ (NSString *)resultSuccess{
    return @"Curiosity:success";
}
// 沙盒是否有指定路径文件夹或文件
+(BOOL)isDirectoryExist:(NSString *)path {
    return [fileManager fileExistsAtPath:path];
}
// 是否是文件夹
+ (BOOL) isDirectory:(NSString *)path{
    BOOL isDir = NO;
    [fileManager fileExistsAtPath:path isDirectory:&isDir];
    return isDir;
}

+(BOOL) isImageFile:(NSString *)path{
    return [path hasSuffix:@".jpg"]
    ||[path hasSuffix:@".png"]
    ||[path hasSuffix:@".JPEG"]
    ||[path hasSuffix:@".JPG"]
    ||[path hasSuffix:@".PNG"];
}

//+ (NSString*)getNetworkStatus:(Reachability*)reachability {
//    if reachability?.isReachableViaWiFi() ?? false {
//      return "wifi"
//    }
//    return "none"
//}

@end

File: example/macos/Runner/Classes/tools/Tools.swift

import Foundation

var curiosity="Curiosity"
let _fileManage=FileManager.init();
public class Tools{
    
    
    func log(info: String) {
//        log(curiosity+"Log:", info)
    }
    
    func nameresultInfo(info: String) -> NSString{
        return curiosity+":"+info
    }
    
    func resultFail() -> NSString {
        return curiosity+":fail"
    }
    
    func resultSuccess() -> NSString {
        return curiosity+":success"
    }
    
    //沙盒是否有指定路径文件夹或文件
    func isDirectoryExist(path: String) ->Bool{
        return _fileManage.fileExists(atPath: path)
    }
    // 是否是文件夹
    func isDirectory(path: String) -> Bool {
        var isDir=false
        _fileManage.fileExists(path, isDirectory:isDir)
        return isDir
    }
    
    func isImageFile(NSString: path) -> bool {
        path.hasSuffix(".jpg")||
        
        return false
    }
    
}
